packages/admin-panel/src/pages/resources/EntityRelationsPage.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TreeResourcePage } from './TreeResourcePage';
+
+export const EntityRelationsPage = ({ getHeaderEl }) => (
+  <TreeResourcePage
+    title="Entity Relations"
+    endpoint="entityHierarchyTree"
+    nodeKey="code"
```

Comment:
In the tradition of `ResoucePage`, I expressed aspects of this resource's config as separate, simple props.

* `endpoint` is the base resource URL that will be used
* `nodeKey` is the field that serves as the identifier for the selected tree and node in the resource URL  (default: `id`)